tests/test_server.py

```diff
@@ -292,6 +293,88 @@ def test_task(self, mock_utcnow):
 
             self.assertEqual(job_result, expected_job_result)
 
+    @unittest.mock.patch('arthur.tasks.datetime_utcnow')
+    def test_job(self, mock_utcnow):
+        """Check whether task data is retrieved"""
+
+        mock_utcnow.return_value = datetime.datetime(2017, 1, 1,
+                                                     tzinfo=dateutil.tz.tzutc())
+
+        with run_server(self.conn) as server:
+            data = {
+                "tasks": [
+                    {
+                        "task_id": "arthur.git",
+                        "backend": "git",
+                        "backend_args": {
+                            "gitpath": os.path.join(self.dir, 'data/git_log.txt'),
+                            "uri": "http://example.com",
+                        },
+                        "category": "commit",
+                        "archive": {},
+                        "scheduler": {
+                            "delay": 10
+                        }
+                    }
+                ]
+            }
+            response = requests.post("http://127.0.0.1:8080/add",
+                                     json=data, headers={'Content-Type': 'application/json'})
+
+            self.assertTrue(response.status_code, 200)
+            self.assertTrue(len(server._tasks.tasks), 2)
+
+            # Run the tasks
+            server.start()
+
+            response = requests.post("http://127.0.0.1:8080/task/arthur.git",
+                                     json=data, headers={'Content-Type': 'application/json'})
+            self.assertTrue(response.status_code, 200)
+
+            task = json.loads(response.content.decode('utf8').replace("'", '"'))
+            jobs = task.pop('jobs')
+
+            # Check jobs data
+            job_task = jobs[0]
+            self.assertEqual(job_task['task_id'], 'arthur.git')
+            self.assertEqual(job_task['job_status'], 'finished')
+
+            job_task_result = job_task['result']
+
+            # Save log in the meta field of the job
```

Comment:
You should explain what you are doing this. In this case, it's because workers don't run in sync mode/testing mode, so logs are not stored into the job.